src/Network/Network.Test/ScenarioTests/NetworkManagerTests.ps1

```diff
@@ -698,4 +886,288 @@ function Test-NetworkManagerResourceMinimumParameterCreate
+    $rgName = Get-ResourceGroupName
+    $networkManagerName = Get-ResourceName
+    $verifierWorkspaceName = Get-ResourceName
+    $rglocation = "eastus2euap"
+    $subscriptionId = "/subscriptions/c9295b92-3574-4021-95a1-26c8f74f8359"
+    $reachabilityAnalysisIntentName = "analysisIntentTest06"
+    $reachabilityAnalysisRunName = "analysisRunTest06"
+
+
+    try{
+        #Create the resource group
+        $resourceGroup = New-AzResourceGroup -Name $rgName -Location $rglocation -Tags @{ testtag = "testval" } 
+
+        # Create Scope
+        $subscriptions  = @($subscriptionId)
+        $scope = New-AzNetworkManagerScope -Subscription $subscriptions
+
+        # Create network manager
+        New-AzNetworkManager -ResourceGroupName $rgName -Name $networkManagerName -NetworkManagerScope $scope -Location $rglocation
+
+        # Create verifier workspace
+        New-AzNetworkManagerVerifierWorkspace -ResourceGroupName $rgName -NetworkManagerName $networkManagerName -Name $verifierWorkspaceName -Location $rglocation -Description "Sample description" 
+
+         # Create analysis intent
+        $sourcePortList = @("100")
+        $destinationPortList = @("99")
+        $protocolList = @("TCP")
+        $sourceIpList = @("192.168.1.10")
+        $destinationIpList = @("172.16.0.5")
+        $groupItem = New-AzNetworkManagerIPTraffic -SourceIps $sourceIpList -DestinationIps $destinationIpList -SourcePorts $sourcePortList -DestinationPorts $destinationPortList -Protocols $protocolList
+
+        $analysisIntent = New-AzNetworkManagerVerifierWorkspaceReachabilityAnalysisIntent -ResourceGroupName $rgName -NetworkManagerName $networkManagerName -VerifierWorkspaceName $verifierWorkspaceName -Name $reachabilityAnalysisIntentName -SourceResourceId "/subscriptions/c9295b92-3574-4021-95a1-26c8f74f8359/resourceGroups/ipam-test-rg/providers/Microsoft.Compute/virtualMachines/testVM" -DestinationResourceId "/subscriptions/c9295b92-3574-4021-95a1-26c8f74f8359/resourceGroups/ipam-test-rg/providers/Microsoft.Compute/virtualMachines/ipam-test-vm-integration-test" -IpTraffic $groupItem
+
+        # Create analysis run
+        # Get the intent ID
+        $intentId = $analysisIntent.Id
+        Write-Host "Analysis Intent ID: $intentId"
+
+        New-AzNetworkManagerVerifierWorkspaceReachabilityAnalysisRun -ResourceGroupName $rgName -NetworkManagerName $networkManagerName -VerifierWorkspaceName $verifierWorkspaceName -Name $reachabilityAnalysisRunName -IntentId $intentId -Description "DESCription"
+
+        # Get analysis run
+        $reachabilityAnalysisRun = Get-AzNetworkManagerVerifierWorkspaceReachabilityAnalysisRun -ResourceGroupName $rgName -NetworkManagerName $networkManagerName -VerifierWorkspaceName $verifierWorkspaceName -Name $reachabilityAnalysisRunName
+        Assert-NotNull $reachabilityAnalysisRun
+        Assert-AreEqual $reachabilityAnalysisRunName $reachabilityAnalysisRun.Name
+
+        Start-TestSleep -Seconds 300
```

Comment:
the only reason to wait here, would be to wait for the analysis run to complete, and for the 'analysisResult' to be populated. Can you add a check for the the analysisResult?